src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2955,4 +2955,10 @@
   <data name="ShowHiddenItems" xml:space="preserve">
     <value>Show hidden items</value>
   </data>
+  <data name="Help" xml:space="preserve">
+    <value>Aide</value>
```

Comment:
Shouldn't this be help?

Reply:
Or maybe it can say "Open documentation"

Reply:
I wrote it in French. It is fixed.